doc/getting_started.rst

```diff
@@ -117,6 +117,17 @@ the test data::
   >>> accuracy_score(pipe.predict(X_test), y_test)
   0.97...
 
+.. note:: Diagram rendering of estimators
+
+    To render estimators as diagrams in notebooks, use the `display='diagram'` option::
+
+    >>> from sklearn import set_config
+    >>> set_config(display='diagram')
+    >>> pipe 
```

Comment:
@thomasjpfan any idea why this does not render as html?

Reply:
This is pure rst, no code is actually ran when the docs are build. We have been linking to a sphinx gallery that properly displays the html repr.

Sidenote: I had a custom sphinx derivative that did just this, but I thought maintaining another sphinx derivative would be too much of a maintenance burden.

Reply:
Should I move the example to another file?

Reply:
Ah right, we do run the docs but not when building them :/

I think we should go with a  dedicated example then, and not change the getting_started guide (https://github.com/scikit-learn/scikit-learn/issues/18305#issuecomment-726770122). @reshamas  I guess we can just merge this PR if you revert the changes made to `getting_started.rst`? Thanks and sorry for the extra work